src/Files.App/Helpers/Navigation/NavigationHelpers.cs

```diff
@@ -293,7 +293,7 @@ private static async Task<FilesystemResult> OpenFile(string path, IShellPage ass
 
 			if (isShortcut)
 			{
-				if (string.IsNullOrEmpty(shortcutInfo.TargetPath))
+				if (string.IsNullOrEmpty(shortcutInfo.TargetPath) && args is not null)
 				{
 					await Win32Helpers.InvokeWin32ComponentAsync(path, associatedInstance, args);
```

Comment:
`InvokeWin32ComponentAsync` actually accepts null, so if you want to get rid of the warning, you should make the args of `InvokeWin32ComponentAsync` null acceptable.